src/Components/Home/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles';
+
+function Home() {
+    return (
+        <View style={styles.home}>
+        <Text style={styles.heading}>INTERESTED IN COMPUTING</Text>

```

Comment:
we need to keep it as a section not in home. Please reuse Section Headers like wiki advice

Reply:
Fixed it now. Can you check?